net/haproxy/src/opnsense/mvc/app/controllers/OPNsense/HAProxy/forms/dialogFrontend.xml

```diff
@@ -154,31 +160,31 @@
         <id>frontend.customOptions</id>
         <label>Option pass-through</label>
         <type>textbox</type>
-        <help><![CDATA[These lines will be added to the HAProxy frontend configuration.<br/><div class="text-info"><b>NOTE:</b> The syntax will not be checked, use at your own risk!</div>]]></help>
+        <help><![CDATA[These lines will be added to the HAProxy frontend configuration.<br/><div class="text-info"><b>NOTE:</b> This is strongly discouraged! The syntax will not be checked, use at your own risk!</div>]]></help>
```

Comment:
Telling users not to do something in a hidden help text does not help. It may be better to ditch this field or simplify the help messages to the minimum.

Reply:
The option pass-through is sort-of a last resort in case a function is missing. I'll try to move these warnings to the "Introduction" page(s).